tests/test_glm_synthetic.py

```diff
@@ -86,25 +86,25 @@ def test_set_up_glm():
     fitted_model.fit(spike_data)
     fit_pred = fitted_model.predict(spike_data)
 
-    fig, axes = plt.subplots(2, 1)
-    axes[0].plot(onp.arange(nt), spike_data[0])
-    axes[0].plot(onp.arange(ws, nt + 1), sim_pred[0])
-    axes[0].plot(onp.arange(ws, nt + 1), fit_pred[0])
-    axes[1].plot(onp.arange(nt), spike_data[1])
-    axes[1].plot(onp.arange(ws, nt + 1), sim_pred[1])
-    axes[1].plot(onp.arange(ws, nt + 1), fit_pred[1])
-    plt.show()
-
-    fig, axes = plt.subplots(nn, nn, sharey=True)
-    for i, j in itertools.product(range(nn), range(nn)):
-        axes[i, j].plot(
-            B.T @ simulated_model.spike_basis_coeff_[i, :, j],
-            label="true"
-        )
-        axes[i, j].plot(
-            B.T @ fitted_model.spike_basis_coeff_[i, :, j],
-            label="est"
-        )
-        axes[i, j].axhline(0, dashes=[2, 2], color='k')
-    axes[-1, -1].legend()
-    plt.show()
\ No newline at end of file
+    # fig, axes = plt.subplots(2, 1)
+    # axes[0].plot(onp.arange(nt), spike_data[0])
+    # axes[0].plot(onp.arange(ws, nt + 1), sim_pred[0])
+    # axes[0].plot(onp.arange(ws, nt + 1), fit_pred[0])
+    # axes[1].plot(onp.arange(nt), spike_data[1])
+    # axes[1].plot(onp.arange(ws, nt + 1), sim_pred[1])
+    # axes[1].plot(onp.arange(ws, nt + 1), fit_pred[1])
+    # plt.show()
+
+    # fig, axes = plt.subplots(nn, nn, sharey=True)
+    # for i, j in itertools.product(range(nn), range(nn)):
+    #     axes[i, j].plot(
+    #         B.T @ simulated_model.spike_basis_coeff_[i, :, j],
+    #         label="true"
+    #     )
+    #     axes[i, j].plot(
+    #         B.T @ fitted_model.spike_basis_coeff_[i, :, j],
+    #         label="est"
+    #     )
+    #     axes[i, j].axhline(0, dashes=[2, 2], color='k')
+    # axes[-1, -1].legend()
+    # plt.show()
```

Comment:
what are these commented out lines? they should either be in a test, deleted, or moved into an issue if it's something we'll eventually want but aren't sure why now.

commented out things get lost and forgotten

Reply:
see earlier comment for plots in tests: I think the problem is `plt.show()`. that does not belong in a test and instead, tests should end with `plt.close('all')`

in this case, I don't think the plots are actually necessary for the tests. but it might be worth including at least one to have a working example for later on?

Reply:
fixed it changing beckend for matplotlib, removed the commented parts